src/template-tags/tickets.php

```diff
@@ -109,6 +109,164 @@ function tribe_events_count_available_tickets( $event = null ) {
 	}
 }//end if
 
+if ( ! function_exists( 'tribe_events_count_tickets_and_rsvp' ) ) {
+
+	/**
+	 * Returns Ticket and RSVP Count for an Event
+	 *
+	 * @param $event_id
+	 *
+	 * @return array
+	 */
+	function tribe_events_count_tickets_and_rsvp( $event_id ) {
```

Comment:
Hrm. Perhaps the logic for `tribe_events_count_tickets_and_rsvp()` should be placed in `Tribe__Tickets__Tickets` as `get_ticket_counts()`. `tribe_tickets_display_count_and_ticket_button()` is pretty specific to display and can probably remain as is.